### ISSUE TITLE
"history_end" bug with default value (was "history_start" relies on heuristics not available in Firefox)

### ISSUE THREAD

**Issue by vermut** [nathan-gs/ha-map-card#34]
Firefox can't parse "4 hours ago". It can parse absolute times I guess but I assume most people would need relative only.
Can you maybe implement boring `hours_to_show` as a fallback for Chrome-only `history_start`?

https://stackoverflow.com/questions/36015877/toisostring-not-working-in-firefox

```
type: custom:map-card
entities:
  - entity: device_tracker.openmower_position
    history_start: 24 hours ago
    history_show_dots: false
x: 56.5
'y': 23.3
```

```
TypeError: end.toISOString is not a function
    subscribe map-card.js:486
    setupHistory map-card.js:334
    render map-card.js:555
    render map-card.js:554
    update lit-element.js:219
    performUpdate updating-element.js:554
    _enqueueUpdate updating-element.js:507
    requestUpdateInternal updating-element.js:473
    set updating-element.js:203
    value hui-section.ts:107
    value hui-section.ts:105
    performUpdate reactive-element.ts:1329
    scheduleUpdate reactive-element.ts:1261
    _$Ej reactive-element.ts:1233
    requestUpdate reactive-element.ts:1208
    set reactive-element.ts:726
    value hui-view.ts:203
    value hui-view.ts:201
    performUpdate reactive-element.ts:1329
    scheduleUpdate reactive-element.ts:1261
    _$Ej reactive-element.ts:1233
    requestUpdate reactive-element.ts:1208
    set reactive-element.ts:726
    value hui-root.ts:513
    _$AE reactive-element.ts:1378
    performUpdate reactive-element.ts:1343
    scheduleUpdate reactive-element.ts:1261
    _$Ej reactive-element.ts:1233
    requestUpdate reactive-element.ts:1208
    set reactive-element.ts:726
    j lit-html.ts:1883
    _$AI lit-html.ts:1825
    v lit-html.ts:1192
    g lit-html.ts:1540
    _$AI lit-html.ts:1384
    z lit-html.ts:2183
    update lit-element.ts:165
    performUpdate reactive-element.ts:1329
    scheduleUpdate reactive-element.ts:1261
    _$Ej reactive-element.ts:1233
    requestUpdate reactive-element.ts:1208
    set reactive-element.ts:726
    value partial-panel-resolver.ts:101
    value hass-router-page.ts:91
    performUpdate reactive-element.ts:1329
    scheduleUpdate reactive-element.ts:1261
    _$Ej reactive-element.ts:1233
    requestUpdate reactive-element.ts:1208
    set reactive-element.ts:726
    j lit-html.ts:1883
    _$AI lit-html.ts:1825
    v lit-html.ts:1192
    g lit-html.ts:1540
    _$AI lit-html.ts:1384
    z lit-html.ts:2183
    update lit-element.ts:165
map-card.js:561:19
    render map-card.js:561
    update lit-element.js:219
    performUpdate updating-element.js:554
    _enqueueUpdate updating-element.js:507
    requestUpdateInternal updating-element.js:473
    set updating-element.js:203
    value hui-section.ts:107
    value hui-section.ts:105
    performUpdate reactive-element.ts:1329
    scheduleUpdate reactive-element.ts:1261
    _$Ej reactive-element.ts:1233
    requestUpdate reactive-element.ts:1208
    set reactive-element.ts:726
    value hui-view.ts:203
    value hui-view.ts:201
    performUpdate reactive-element.ts:1329
    scheduleUpdate reactive-element.ts:1261
    _$Ej reactive-element.ts:1233
    requestUpdate reactive-element.ts:1208
    set reactive-element.ts:726
    value hui-root.ts:513
    _$AE reactive-element.ts:1378
    performUpdate reactive-element.ts:1343
    scheduleUpdate reactive-element.ts:1261
    _$Ej reactive-element.ts:1233
    requestUpdate reactive-element.ts:1208
    set reactive-element.ts:726
    j lit-html.ts:1883
    _$AI lit-html.ts:1825
    v lit-html.ts:1192
    g lit-html.ts:1540
    _$AI lit-html.ts:1384
    z lit-html.ts:2183
    update lit-element.ts:165
    performUpdate reactive-element.ts:1329
    scheduleUpdate reactive-element.ts:1261
    _$Ej reactive-element.ts:1233
    requestUpdate reactive-element.ts:1208
    set reactive-element.ts:726
    value partial-panel-resolver.ts:101
    value hass-router-page.ts:91
    performUpdate reactive-element.ts:1329
    scheduleUpdate reactive-element.ts:1261
    _$Ej reactive-element.ts:1233
    requestUpdate reactive-element.ts:1208
    set reactive-element.ts:726
    j lit-html.ts:1883
    _$AI lit-html.ts:1825
    v lit-html.ts:1192
    g lit-html.ts:1540
    _$AI lit-html.ts:1384
    z lit-html.ts:2183
    update lit-element.ts:165
```

**Comment by poisondima** [comment]
I was getting the same error until I specified the ending:
```
           history_start: ‘24 hours ago’
           history_end: now
```

**Comment by nathan-gs** [comment]
@vermut you are missing the `history_end` part, so it's not a Firefox issue (also my daily browser). @poisondima indeed!

However since [v0.8.2](https://github.com/nathan-gs/ha-map-card/releases/tag/v0.8.2) if `history_end` is missing, it should default to `now`.

**Comment by nathan-gs** [comment]
Ok, there is a little bug in `history_end` with `now` as default, so will fix this.

**Comment by nathan-gs** [comment]
Fixed in [v0.9.1](https://github.com/nathan-gs/ha-map-card/releases/tag/v0.9.1)
